course_catalog/etl/youtube_test.py

```diff
@@ -101,32 +100,31 @@ def mock_video_batch_data(mock_video_data):
     return data
 
 
-@pytest.fixture
-def mock_channel_data():
-    """Mock video channel yml"""
-    return """
----
-channels:
-  - offered_by: MIT
-    channel_id: channel1
-    playlists:
-      - playlist1
-  - offered_by: OCW
-    channel_id: channel2
-    playlists:
-      - playlist2
-"""
+def mock_channel_file(offered_by):
+    """Mock video channel github file"""
+
+    content = """
+offered_by: {offered_by}
+channel_id: channel
+playlists:
+  - id: playlist
+""".format(
+        offered_by=offered_by
+    )
+
+    mocked_github_file = MagicMock()
```

Comment:
I think this only needs to be a regular `Mock()`, you can also make it a one-liner:

```python
return Mock(decoded_content=content)
```